3001-4000/LIT3326Melchiz1.xml

```diff
@@ -40,7 +55,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
-         <langUsage><language ident="en">English</language></langUsage>
+         <abstract><p>The homily on Melchizedek's origin attributed to <persName ref="PRS3255CyrilI"/>, mainly circulate as part of the theological collection 'Qərəllos'. It was certainly translated from the Greek: the original is lost and the Ethiopic version is the only existing one. 
```

Comment:
Maybe refer to Qərəllos with encoding?

And I think we need to credit a source with the idea that this work was certainly translated from Greek (if it is not your own opinion, but then it would also be good to signal this)

Reply:
Good idea!  In case you think my changes are ok, I will use it as a model for further abstracts then.